client/app/visualizations/chart/plotly.js

```diff
@@ -514,9 +517,11 @@ const CustomPlotlyChart = (clientConfig) => {
           });
         });
       };
-      scope.$watch('options.customCode', () => {
+      scope.$watch('[options.customCode, options.autoRedraw]', () => {
```

Comment:
When I am editing a small script, I would like to refresh graph as I type. This is done by watching `customCode`.
When I am editing a big script, I would like to turn off the refresh-as-I-type, but when I click on the auto redraw checkbox, I want so see the graph refreshed immediately. This last part is done by watching `autoRedraw`.